Platforms/Apple/src/AppleDebug.mm

```diff
@@ -58,7 +58,7 @@
             }
             else
             {
-                return false;
+                return (signed char)false;
```

Comment:
I think this change made it to PR by accident.

Reply:
It's unrelated, but I build on OSX too and this was needed for the latest clang.

Reply:
Will remove this if you want. Otherwise I don't see the point of a separate pull request.